Packs/CheckpointFirewall/Playbooks/Checkpoint_-_Publish&Install_configuration_README.md

```diff
@@ -0,0 +1,38 @@
+Publish the Checkpoint Firewall configuration and install policy over all the gateways are available.
+Please note that only CheckPoint admin users can run this playbook.
```

Comment:
Isn't the integration using an admin privilege API key anyway?

Reply:
Not necessarily, there is a separate list for API users, which means that a user can be given access without admin privileges to perform certain actions through the API.